strawberry/custom_scalar.py

```diff
@@ -19,6 +20,10 @@ class ScalarDefinition(StrawberryType):
     parse_value: Optional[Callable]
     parse_literal: Optional[Callable]
 
+    # Optionally store the GraphQLScalarType instance so that we don't get
+    # duplicates
+    implementation: Optional[GraphQLScalarType] = None
```

Comment:
we should find a way to get rid of this, as we don't want to couple Strawberry types to GraphQL-core, but we can do this in another PR :)

Reply:
Yeah it’s not ideal…

Reply:
I think when we introduce StrawberryScalar we can change this :)